docs/swagger.yaml

```diff
@@ -1098,10 +1098,8 @@ paths:
         - in: header
           name: Authorization
           description: >-
-            The access token is a base64-encoded string that is required for all API calls.
```

Comment:
This comes from the login api access token description. The token used here is not the one that we get from the accesstoken/get endpoint

Reply:
are you sure? I thought we changed this to use the normal ecom auth

Reply:
That used to be the case but this changed about 12 months ago
It now used normal ecom token

Reply:
@tht13 - Can we look at the postman together this week and get it set up correctly?